### DISCUSSION TITLE
How to subset the project according to the chromosome number?

### DISCUSSION THREAD

**Discussion by woshiyangsi** [GreenleafLab/ArchR#1403] [General]
Hi ArchR team,
How to subset the project according to the chromosome number,
For example, I want to extract chromosome 1 from the project

**Comment by rcorces** [comment]
thats not currently possible. Even if you went back to the start and dropped all other chromosomes from your fragment files, you will likely run into issues because chromosomes have 0 reads. We are actively trying to fix that 0 read issue.

**Comment by woshiyangsi** [comment]
Thank you for your quick reply.I have two more questions to ask.
1. What is the format of the region when drawing according to the region in the plotbrowsertrack() function? I tried region = "1:100000-200000", but I reported an error.
2. If I want to personalize the drawing of browsertrack, am I based on proj@peakSet to draw? If not, how can I personalize the drawing?

**Reply by rcorces** [reply]
1. Read the function documentation - https://www.archrproject.com/reference/plotBrowserTrack.html
2. Please clarify

**Comment by woshiyangsi** [comment]
ok, as for the second question,
For example, I want to draw the peak browser track of multiple genes on one map (As shown in the figure below), 
Therefore, I want to extract data from proj and draw it with other R packages. What information should I extract from proj to draw?
![image](https://user-images.githubusercontent.com/54662832/165437045-d24b6727-e62d-42a7-a75a-100d88b959e8.png)

**Reply by rcorces** [reply]
I would use `getGroupBW()` to export bigWig files for plotting in other packages.
https://www.archrproject.com/reference/getGroupBW.html

**Comment by woshiyangsi** [comment]
Ok, i got it, thanks.

**Reply by woshiyangsi** [reply]
hi @rcorces ,
There is a peak in the track area, but there is no peak in the peak area. How to understand this phenomenon?
For example ASMTL.
![image](https://user-images.githubusercontent.com/54662832/165660437-242b981d-ca30-4dd8-9d1d-6782d4b4e2a9.png)

**Reply by rcorces** [reply]
The peaks that are retained during `addReproduciblePeakSet()` are dependent on the filter that you set in that function for peak significance and for peak reproducibility. This peak must not have passed those thresholds.

**Comment by woshiyangsi** [comment]
Thanks for your reply. Another question I'm curious about is, when building the peak set, what are your considerations to exclude the peak on the Y chromosome by default?

**Reply by rcorces** [reply]
Best to ask different questions in a separate discussion post so that they are easily searchable by other users.
But to answer your question, if you all chrY peaks, then all XY individuals will have signal in regions not present in XX individuals, making these samples appear artificially more similar to each other. This is quite standard practice when analyzing samples from individuals of differing biological sex.

**Reply by woshiyangsi** [reply]
OK, thanks
